### PR TITLE
Rb 4.0.x recalboxsupport test

### PR DESCRIPTION
**Pull request by Laddyacris**
Please make sure your PR is ready to be merged !
- [X] You added the changes in CHANGELOG.md
- [X] You choose the right repository branch to make the PR
- [X] You described the PR as below

Fixes #XXXX

Changes :
- Add ifconfig -a for network
## \- Add recalbox-boot.conf for debug external usb storage

Related to (put here the others PR in other repositories)

### CODE REVIEW

File: board/recalbox/fsoverlay/recalbox/scripts/recalbox-support.sh [hunk 1]
```diff
@@ -47,6 +47,7 @@ netstat -tuan    > "$DSYSTEM""/netstat.txt"
 lsusb -v         > "$DSYSTEM""/lsusb.txt" 2>/dev/null
 tvservice -m CEA > "$DSYSTEM""/tvservice-CEA.txt"
 tvservice -m DMT > "$DSYSTEM""/tvservice-DMT.txt"
+ifconfig -	 > "$DSYSTEM""/ifconfig.txt"
```
Comment (digitalLumberjack) [inline board/recalbox/fsoverlay/recalbox/scripts/recalbox-support.sh]: ifconfig - ne doit par marcher, tu voulais mettre ifconfig -a ?

**Comment by Laddyacris** [comment]
arf oui grrr

**Comment by Laddyacris** [comment]
I think is ok now

**Comment by digitalLumberjack** [comment]
Thanks

**Pull request merged**